### ISSUE TITLE
AoE looting causes issues with quest drops and non stackable items like quest starters or recipes

### ISSUE THREAD

**Issue by EdyGlockenspiel** [gamefreedomgit/Maelstrom#3070]
**Description:**
I am repurposing this bug report as the actual main issue here is AoE looting. Many reports on discord in the support channel have shown stuff like recipes, quest starters like battered hilt and quest items for basic quests not dropping properly in AoE looting. A common error message is "You have looted that item already" despite clearly not. Because of this, AoE looting is actually a *hinderance* rather than a QoL improvement. I always now loot mobs individually as I am killing them to make sure I go around the bug. I shouldn't have to do that. Combat looting prolongs combat and can lead to your death as well. My suggestion would be to just remove AoE looting as a whole. Additionally, when you kill a LOT of mobs, lets say 15+, AoE looting seems to disable itself and you still need to loot most of them individually anyway. 

**How to reproduce:**
Kill a bunch of mobs that can drop quest items, quest starters or recipes ect and try to loot them. This affects MANY mobs in pre cata areas, this includes the mobs that drop the battered hilt.

**How it should work:**
This one is subjective, as AoE looting was not added until the pre patch for WoD, so technically it should not be in cata at all. But for WoD and beyond servers, I am not exactly sure the mechanics/scripts behind it, but I can safely say I never had problems looting mobs with AoE looting. If I had to take a guess, once you open the loot window for a mob, it compiles all of the loot for all in range lootable mobs and merges them together into one giant loot window, merging all the gold, stackable items ect to one stack of each and any non stackables are at the bottom with some times several pages of loot from the one mob you actually looted if they all dropped different stuff.

**Database links:**
https://cata-twinhead.twinstar.cz/?npc=25721
https://cata-twinhead.twinstar.cz/?item=35493#dropped-by
https://cata-twinhead.twinstar.cz/?quest=11914
Keeping this quest's links here as reference to a one of the mobs I had problem with, but there are many.

**Comment by EdyGlockenspiel** [comment]
So I am almost 100% sure this is an AoE looting problem now. Remembering back to when I wrote this, every pack of mobs I gathered and AoEed down dropped exactly one of the items. I was questing in storm peaks just now and needed to get mammoth meat for a quest, every individual mob I looted dropped it, but when I killed 3 together I only got one. So this tells me this is a problem with AoE looting as a whole.
